packages/ses/src/whitelist.js

```diff
@@ -1644,6 +1645,19 @@ export const whitelist = {
     '@@toStringTag': 'string',
   },
 
+  // TODO: To be replaced with Promise.delegate
```

Comment:
Let’s leave more note here:
HandledPromise is a temporary non-standard affordance for the Agoric runtime.
We intend to change this to `%HandledPromise%`, not revealed on `globalThis` but returned by `Promise.delegate`, which is not yet implemented.

Or words that are closer to true.